source/deprecations/v1.x.html.md

```diff
@@ -349,6 +349,13 @@ making the content a `SafeString`. For example:
   })
 ```
 
+Make sure you don't put quotes around the string marked-as-safe when you bound
+it in the template, as this would prevent Ember from seeing it as safe:
+
+```handlebars
+<div style={{myStyle}}></div>
```

Comment:
True, I've moved that paragraph under the definition of the `myStyle` property and removed the duplicated template code snippet.